src/main/java/com/icthh/xm/tmf/ms/activation/service/SagaSpecService.java

```diff
@@ -64,7 +66,11 @@ private void refreshConfig(String updatedKey, String config) {
     }
 
     private void updateRetryPolicy(SagaSpec spec) {
-        spec.getTransactions().forEach(tx -> tx.getTasks().forEach(task -> task.applyAsDefaultTransactionConfig(tx)));
+        spec.getTransactions().forEach(tx -> tx.setTasks(
```

Comment:
add test please

Reply:
Actually, this is exactly the code fix, because the tests in the master are failed